### PR TITLE
Intersection Point of Y shaped LinkedList

### PR DESCRIPTION
**Pull request by Rishikeshrajrxl**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(As I was suggested to make one PR for one issue at a time. So I have made this new PR.
I have made this program user interactive with a suitable interface. 
Also, the input is based on the user and how they want to intersect to which point for cross-validation.
 Please review it and let me know if any changes needed.)

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#495 )

### CODE REVIEW

**Comment by Rishikeshrajrxl** [comment]
@plazzy99, I have made new PR having only one issue, do review this.

**Review by vatsalkesarwani12 — APPROVED** [review]
Great work @Rishikeshrajrxl 
your code flows like documentation and it has a high level of readability 
Impressive 💯

**Comment by Rishikeshrajrxl** [comment]
> Great work @Rishikeshrajrxl
> your code flows like documentation and it has a high level of readability
> Impressive

Thanks @plazzy99 , it means a lot.

File: C-Plus-Plus/cp/Intersection Point of Y shaped LinkedList.cpp
Comment (atarax665) [inline C-Plus-Plus/cp/Intersection Point of Y shaped LinkedList.cpp]: Add sample input/output in commented form, rest everything looks fine.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/cp/Intersection Point of Y shaped LinkedList.cpp
Comment (Rishikeshrajrxl) [reply C-Plus-Plus/cp/Intersection Point of Y shaped LinkedList.cpp]: Done adding, Please review again.

**Comment by Rishikeshrajrxl** [comment]
@Kajol-Kumari 
@Abhijit2505 
Please review this assigned work.

**Review by Abhijit2505 — APPROVED** [review]
Looking Good To Me

**Comment by Rishikeshrajrxl** [comment]
@siddharth25pandey, it has been a long time and I have got 2 approval and still waiting for one more.
please review it and kindly merge it.

**Comment by Rishikeshrajrxl** [comment]
@Kajol-Kumari 
waiting for your review.

**Comment by siddharth25pandey** [comment]
> @siddharth25pandey, it has been a long time and I have got 2 approval and still waiting for one more.
> please review it and kindly merge it.

@Rishikeshrajrxl Already mentors were assigned. They will review it shortly

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Comment by Rishikeshrajrxl** [comment]
@atarax665 
Needed your last review, please do this and merge it.

**Review by atarax665 — APPROVED** [review]

**Pull request merged**
